packages/headless/src/features/configuration/configuration-actions.ts

```diff
@@ -135,7 +143,11 @@ export const updateAnalyticsConfiguration = createAction(
       enabled: new BooleanValue({default: true}),
       originLevel2: originSchemaOnConfigUpdate(),
       originLevel3: originSchemaOnConfigUpdate(),
-      apiBaseUrl: new StringValue({url: true, emptyAllowed: false}),
+      apiBaseUrl: new StringValue({
+        relativeUrl: true,
+        url: true,
+        emptyAllowed: false,
+      }),
```

Comment:
Hi Tom, why should the `platformUrl` and `apiBaseUrl` options support relative values? Could you provide more context behind what is driving this change?

Reply:
Makes sense, thanks